src/main/java/org/opentripplanner/updater/vehicle_rental/GBFSMappings/GbfsResponse.java

```diff
@@ -9,6 +11,7 @@
  * Response class for the gbfs.json file.
  * See https://github.com/NABSA/gbfs/blob/master/gbfs.md#gbfsjson
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Could this be applied to the `BaseGtfsResponse` and have a cascading effect to other mappings that extend this class?